### DISCUSSION TITLE
Vertical display-Line chart dates

### DISCUSSION THREAD

**Discussion by debjanipaul** [h2oai/wave#1548] [Q&A]
Hi I would like to know if its possible to display the dates vertically or  diagonally instead of horizontally for the ui.plot line charts, as it gets too crowded. Or can we limit the number of labels in the x-axis e.g 5 or 8 date labels?

![linechart](https://user-images.githubusercontent.com/23644204/179264568-42cbb524-dfb7-4331-8c36-2eadccb5bd5d.PNG)

```
ui.visualization(
                            height='300px',
                            plot=ui.plot([ui.mark(type='line', x='=created_dt', y='=count', color='=count',
                                           color_domain=['#007acc'], color_range='#007acc')]),
                            data=data(fields='created_dt count', columns=[df[col].to_list() for col in df.columns], pack=True)
                    )

**Comment by mturoci** [comment]
Added in da7f02f12ceb16d945fcb0ca076bea178b2c3a57. Will be available in `0.23` release. For now, you can play with it using nightly if you wish.
